modules/virt-cloning-a-datavolume.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-cloning-a-datavolume-using-smart-cloning.adoc
+
+[id="virt-cloning-a-datavolume_{context}"]
+= Cloning a DataVolume
+
+To clone a DataVolume (DV), do the following:
+
+* Create a DV with a PersistentVolumeClaim (PVC) source. See the Additional Resources section for more information about
```

Comment:
Could you provide the actual command(s) for creating the DV?

This existing one gives the DV object YAML to save as a file, and then the oc command to create the DV based on the contents of that file: https://cnv-1396--ocpdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/cloning_vms/virt-cloning-vm-disk-into-new-datavolume.html#virt-cloning-pvc-of-vm-disk-into-new-datavolume_virt-cloning-vm-disk-into-new-datavolume

I see that you're linking to it in the additional resources, but the procedure is so short that I would include the steps here instead of trying to link to it. (Especially since we can't xref in modules at the moment).

Reply:
Replaced with steps. This is always a hard decision for me (mostly since I've come from environments where anything can be easily single-sourced). But if you're good with it, so am I.

Reply:
Yeah understandable. If we could link in the middle of a procedure, I would say that it would be fine to not duplicate. But since we have that restriction, I think it'll help readers to have the steps right inline here.